students/viveksyngh/quiz.go

```diff
@@ -0,0 +1,94 @@
+package main
+
+import (
+	"encoding/csv"
+	"os"
+	"fmt"
+	"strings"
+	"log"
+	"flag"
+	"time"
+	"math/rand"
+) 
+
+type Question struct {
```

Comment:
It's a good idea to document an exported struct. You can use `golint` tool to tell you about these actions:
```bash
golint students/viveksyngh
```